core/plugin/builtin_plugins/chat/main.py

```
@@ -67,27 +74,42 @@ async def handle_msg(self, event: KiraMessageEvent):
+                    return
+                try:
+                    await self.ctx.message_processor.flush_session_messages(sid)
+                except Exception:
+                    logger.exception(f"[Debounce] Error flushing session {sid}")
+                return
+        except asyncio.CancelledError:
+            raise
+        finally:
+            current_task = asyncio.current_task()
+            if self.session_tasks.get(sid) is not current_task:
+                return
```

Ruff (0.16.2)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)